arbitrum-docs/build-decentralized-apps/token-bridging/bridge-tokens-programmatically/02-how-to-bridge-tokens-standard.mdx

```diff
@@ -94,7 +94,7 @@ Now, here’s an explanation of the contracts and methods that need to be called
 - When bridging from Ethereum (L1) to Arbitrum (L2), you’ll need to interact with the `L1GatewayRouter` contract, by calling the `outboundTransferCustomRefund` method. This router contract will relay your request to the appropriate gateway contract, in this case, the `L1ERC20Gateway` contract. To get the address of the gateway contract that’s going to be used, you can call the `getGateway` function in the `L1GatewayRouter` contract.
 - When bridging from Arbitrum (L2) to Ethereum (L1), you’ll need to interact with the `L2GatewayRouter` contract, by calling the `outBoundTransfer` method. This router contract will relay your request to the appropriate gateway contract, in this case, the `L2ERC20Gateway` contract. To get the address of the gateway contract that’s going to be used, you can call the `getGateway` function in the `L2GatewayRouter` contract.
 
-You can find the addresses of the contracts involved in the process in [this page](/build-decentralized-apps/reference/02-useful-addresses.md#token-bridge-smart-contracts).
+You can find the addresses of the contracts involved in the process in [this page](/build-decentralized-apps/reference/02-contract-addresses.mdx).
```

Comment:
This diff removes the anchor that was present before
`#token-bridge-smart-contracts`

Reply:
This is a limitation of Docusaurus: it cannot process anchor links within partials. Yet.

Reply:
I think I solved the issue. Please check the deployment to verify.